rq/worker.py

```diff
@@ -668,20 +681,39 @@ def unsubscribe(self):
             self.pubsub.unsubscribe()
             self.pubsub.close()
 
-    def reorder_queues(self, reference_queue):
-        """Method placeholder to workers that implement some reordering strategy.
-        `pass` here means that the queue will remain with the same job order.
+    def reorder_queues(self, reference_queue: 'Queue'):
+        """Reorder the queues according to the strategy.
+        As this can be defined both in the `Worker` initialization or in the `work` method,
+        it doesn't take the strategy directly, but rather uses the private `_dequeue_strategy` attribute.
 
         Args:
-            reference_queue (Union[Queue, str]): The queue
+            reference_queue (Union[Queue, str]): The queues to reorder
         """
-        pass
+        if self._dequeue_strategy is None:
+            self._dequeue_strategy = DequeueStrategy.DEFAULT
+
+        if self._dequeue_strategy not in ["default", "random", "roundrobin"]:
```

Comment:
I'm ok with not doing anything if `DequeueStrategy` is not round robin or random.

Reply:
ok

Reply:
Double checking, we don't do anything, there's just a warning here, since it means someone is using an argument that is not known, so it won't work as the user expected. I would favor keeping the log. Let me know if there are any further actions here.

Reply:
Marking as resolved, let me know if you'd like to change this.